skycultures/modern/index.json

```diff
@@ -1675,6 +1675,53 @@
       "common_name": {"english": "Sails", "native": "Vela"}
     }
   ],
+  "zodiac": { "name": { "native": "Signa Zodiacala", "english": "Zodiacal signs"},
+              "partitions": [ 12, 30],
+              "extent": 8,
+	      "names": [ { "symbol": "\u2648", "native": "Aries", "english": "Ram"},
+			 { "symbol": "\u2649", "native": "Taurus", "english": "Bull"},
+			 { "symbol": "\u264A", "native": "Gemini", "english": "Twins"},
+			 { "symbol": "\u264B", "native": "Cancer", "english": "Crab"},
+			 { "symbol": "\u264C", "native": "Leo", "english": "Lion"},
+			 { "symbol": "\u264D", "native": "Virgo", "english": "Virgin"},
+			 { "symbol": "\u264E", "native": "Libra", "english": "Scales"},
+			 { "symbol": "\u264F", "native": "Scorpio", "english": "Scorpion"},
+			 { "symbol": "\u2650", "native": "Capricornus", "english": "Capricorn"},
+			 { "symbol": "\u2651", "native": "Sagittarius", "english": "Archer"},
+			 { "symbol": "\u2652", "native": "Aquarius", "english": "Water bearer"},
+			 { "symbol": "\u2653", "native": "Pisces", "english": "Fishes"}]},
+  "lunar_system":  { "name": "Nakshatras",
+  "partitions": [ 27, 4],
+              "extent": 5,
+	      "link": { "star": 65474, "offset": 180},
+	      "names": [ { "symbol": "LS01", "native": "अश्विनी",             "pronounce": "Ashvini",    "english": "Physician to the Gods"},
+			 { "symbol": "LS02", "native": "भरणी",             "pronounce": "Bharani",    "english": "Bearer"},
+			 { "symbol": "LS03", "native": "कृत्तिका",           "pronounce": "Krittika",    "english": "Pleiades"},
+			 { "symbol": "LS04", "native": "रोहिणी",            "pronounce": "Rohini",     "english": "Red One"},
+			 { "symbol": "LS05", "native": "मृगशिर",            "pronounce": "Mrigashira", "english": "Deer's Head"},
+			 { "symbol": "LS06", "native": "आर्द्रा",             "pronounce": "Ardra",      "english": "Storm God"},
+			 { "symbol": "LS07", "native": "पुनर्वसु ",            "pronounce": "Punarvasu",  "english": "Two Restorers of Goods"},
+			 { "symbol": "LS08", "native": "पुष्य",              "pronounce": "Pushya",    "english": "Nourisher"},
+			 { "symbol": "LS09", "native": "आश्ळेषा/आश्लेषा",       "pronounce": "Āshleshā",   "english": "Embrace"},
+			 { "symbol": "LS10", "native": "मघा",              "pronounce": "Maghā",     "english": "Bountiful"},
```

Comment:
please make it consistently (either all with or all without diacritics)

Reply:
30 minutes vs. 6 hours. Yes, later.

Reply:
https://xing.fmi.uni-jena.de/mediawiki/index.php/Indian_Names_(All_Terms)

Reply:
To make it consistent, should I use the column with or without diacritics? Look in LS2, and a few others: You list totally different names in the columns. How do I know what's the correct transliteration of the Sanskrit? You know what: I let you fix the strings. :-) I moved the block into Indian SC now.

Reply:
@vvasuki provided the strings now.